app/routes/verkiezingen/verkiezingsuitslag/prepare.js

```diff
@@ -1,9 +1,60 @@
 import Route from '@ember/routing/route';
 
+import { service } from '@ember/service';
+
 export default class PrepareInstallatievergaderingRoute extends Route {
-  async model() {
+  @service store;
+
+  queryParams = {
+    filter: { refreshModel: true },
+    sort: { refreshModel: true },
+  };
+
+  async model(params) {
+    const verkiezingsuitslag = this.modelFor('verkiezingen.verkiezingsuitslag');
+    const bestuursorgaan =
+      await verkiezingsuitslag.installatievergadering.bestuursorgaanInTijd;
+
+    let mandatarissen;
+    if (bestuursorgaan) {
+      mandatarissen = await this.store.query(
+        'mandataris',
+        this.getOptions(params, bestuursorgaan)
+      );
+    }
+
     return {
-      ...this.modelFor('verkiezingen.verkiezingsuitslag'),
+      ...verkiezingsuitslag,
+      mandatarissen: mandatarissen,
     };
   }
+
+  getOptions(params, bestuursOrgaan) {
```

Comment:
This needs to be reviewed when we construct the right tables, but for now this is good for testing the table functionalities.